pkg/kubectl-argo-rollouts/cmd/resume/resume.go

```diff
@@ -29,7 +29,7 @@ func NewCmdResume(o *options.ArgoRolloutsOptions) *cobra.Command {
 			}
 			rolloutIf := o.RolloutsClientset().ArgoprojV1alpha1().Rollouts(o.Namespace())
 			for _, name := range args {
-				ro, err := rolloutIf.Patch(name, types.MergePatchType, []byte(`{"spec":{"paused":false}}`))
+				ro, err := rolloutIf.Patch(name, types.MergePatchType, []byte(`{"status":{"pauseConditions":null}}`))
```

Comment:
A resume action should set spec.paused to false as well as nill out the conditions. As a user, I also want to use this CLI to resume something I paused myself.